ClassTranscribeServer/Controllers/AdminController.cs

```diff
@@ -182,7 +182,12 @@ public async Task<IActionResult> GetLogs(DateTime from, DateTime to)
                 await stream.CopyToAsync(memory);
             }
             memory.Position = 0;
-            return File(memory, "text/csv", Path.GetFileNameWithoutExtension(path) + ".csv");
+
```

Comment:
This code seems to have some convoluted IO; create a file (path). Create a memory stream; read it again to make another file. Is it possible to stream the output directly without creating the whole thing into memory?